data/app-template.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
Please don't delete this file.

Reply:
Sorry about that, saw that it wasn’t needed in the gmail setup I was following. How can I go about merging this back in? Copy and Paste? If so, at what line should it be pasted in. Truly sorry if I am messing things up, learning how to do this.

Reply:
Just a file with the same name and the same content.